api/myth/definition.proto

```diff
@@ -66,6 +66,8 @@ message QuoteMythTokenWithdrawalResponse {
     string total_amount = 1;
     /* Gas cost of withdrawal */
     string gas_fee = 2;
+    int64 quote_id = 3;
```

Comment:
How about making this a string so we can use a UUID for the quote id?

Reply:
I have changed this to String now.